Add UIAPduino Pro Micro CH32V003

YuukiUmeta-UIAP opened the pull request:
That is hardware implementation inspired by cnlohr/rv003usb

YuukiUmeta-UIAP commented:
Sorry, the directory name was in the wrong case letter.

I have been fixed.

tannewt reviewed (changes requested):
Where are the hardware files provided with a license? I see a tar download but it only has the eagle files. Is there a hardware repo with a license?

YuukiUmeta-UIAP commented:
> Where are the hardware files provided with a license? I see a tar download but it only has the eagle files. Is there a hardware repo with a license?

That is the eagle files. Those files are include references of license URL. Aren,t enough?

tannewt commented:
> That is the eagle files. Those files are include references of license URL. Aren,t enough?

I don't have Eagle so it's better to have it outside the file. (Usually folks have a git repo with a license file separate.)

YuukiUmeta-UIAP commented:
> > That is the eagle files. Those files are include references of license URL. Aren,t enough?
> 
> I don't have Eagle so it's better to have it outside the file. (Usually folks have a git repo with a license file separate.)

Thank you for your suggestion. I moved it to github repository and added license file.

tannewt reviewed (approved):
Thank you!
